courses/free-courses-en.md

```diff
@@ -808,6 +808,7 @@
 * [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) - Osio Labs Inc. *(signup requested, not required)*
 * [Express JS Crash Course](https://www.youtube.com/watch?v=L72fhGm1tfE) - Brad Traversy, Traversy Media (YouTube)
 * [ExpressJS Fundamentals](https://www.udemy.com/course/expressjs-fundamentals/) - Patrick Schroeder (Udemy)
+* [Node / Express Course](https://www.youtube.com/watch?v=qwfE7fSVaZM) - Freecodecamp (Youtube)
```

Comment:
```suggestion
* [Node.js / Express Course - Build 4 Projects](https://www.youtube.com/watch?v=qwfE7fSVaZM) - Freecodecamp (Youtube)
```
Don't invent titles! Use the title given by the resource.
Is this the same as the John Smilga, Coding Addict course in the next listing?

Reply:
Sorry my bad.Deleting my PR